test/e2e/provisioning/migrateuidscenario.go

```diff
@@ -145,15 +145,22 @@ func verifyMigrateUID(kubeConfig, manifestPath string, parameters []string, time
 
 	// Step 4: Delete the instance and then verify instance is gone
 	for i := 0; i < maxTries; i++ {
+
 		// Deletion part 0: Delete and continue on err if there are tries left
-		if err := prov.Delete(machine, machineCreateDeleteData); err != nil {
+		done, err := prov.Cleanup(machine, machineCreateDeleteData)
+		if err != nil {
 			if i < maxTries-1 {
 				glog.V(4).Infof("Failed to delete machine %s on try %v with err=%v, will retry", machine.Name, i, err)
 				time.Sleep(10 * time.Second)
 				continue
 			}
 			return fmt.Errorf("failed to delete machine %s: %v", machine.Name, err)
 		}
+		if !done {
+			// Resources are not gone yet, check in a few seconds again
+			time.Sleep(30 * time.Second)
```

Comment:
`30 seconds == a few seconds`..?

Reply:
I set it to 10 seconds as it seems to be the common wait period in the cmd.
Also i updated the comment.

https://github.com/kubermatic/machine-controller/pull/385/commits/90f5bfd702b525c9ccbf0da106bc5ae30a5c8a66